nostalgia/ndf.py

```diff
@@ -360,7 +360,7 @@ def create_sample_data(self):
         fname = os.path.join(nostalgia_dir, "data/samples/" + self.df_name + ".parquet")
         # verify that we can process it
         _ = self.as_simple()
-        sample = self.iloc[:100].reset_index().drop("index", axis=1)
+        sample = self.iloc[:1000].reset_index().drop("index", axis=1)
```

Comment:
I presume this increases the amount of sample data by a factor of 10?  (Assuming that much exists.)

Reply:
Look at the next few lines in which we `sample`. It's more that we sample from a larger part of the dataframe.

I think the actual unsolved problem now that I look at it, is that we are trying to validate the data with `as_simple` the line before, but this can be super slow if you have a lot of data. So taking a slice is a little too late anyway.

Reply:
> Look at the next few lines in which we sample. It's more that we sample from a larger part of the dataframe.

Ah, I see.  That's helpful.

> I think the actual unsolved problem now that I look at it, is that we are trying to validate the data with `as_simple` the line before, but this can be super slow if you have a lot of data. So taking a slice is a little too late anyway.

Could this be fixed by making a copy with a smaller amount of data before validating?